frontend/src/components/nodes/NodeRow.tsx

```diff
@@ -32,7 +33,9 @@ export default class extends React.PureComponent<Props> {
         <Col md="7" xs="7">
           <Row>
             <Col className="node-row-title">
-              @{node.accountId}{" "}
+              @{node.accountId}
+              {"   "}
+              <span>${node.stake && <Balance amount={node.stake} />}</span>
```

Comment:
1. I think we don't need that `$` sign
2. I would like to have some indication that it is a stake, not an account balance or anything else. @corwinharrell @kcole16 any suggestions on how to make this clear?

Reply:
Something like "Staking 100,000 N"?

Reply:
@kcole16 keep it stupid simple! I love it! :smile: